imap_processing/hi/l1a/science_direct_event.py

```diff
@@ -283,13 +285,31 @@ def create_dataset(de_data_list: list, packet_met_time: list) -> xr.Dataset:
         trigger_id, dims="epoch", attrs=hi_cdf_attrs.trigger_id_attrs.output()
     )
     dataset["tof_1"] = xr.DataArray(
-        tof_1, dims="epoch", attrs=hi_cdf_attrs.tof_attrs.output()
+        tof_1,
+        dims="epoch",
+        attrs=dataclasses.replace(
+            hi_cdf_attrs.tof_attrs,
+            fieldname="Time of Flight(TOF) 1",
```

Comment:
Do we want a space between Flight / parentheses?
```suggestion
            fieldname="Time of Flight (TOF) 1",
```